tests/config/drone/check_web_cache.sh

```diff
@@ -9,17 +9,13 @@ if [ -z "$1" ]; then
 fi
 
 echo "Checking web version - $WEB_COMMITID in cache"
+web_cache=$(mc find s3/$CACHE_BUCKET/ocis/web-test-runner/$WEB_COMMITID/$1 2>&1 | grep 'Object does not exist')
 
-URL="$CACHE_ENDPOINT/$CACHE_BUCKET/ocis/web-test-runner/$WEB_COMMITID/$1.tar.gz"
-
-echo "Checking for the web cache at '$URL'."
-
-if curl --output /dev/null --silent --head --fail "$URL"
-then
-	echo "$1 cache with commit id $WEB_COMMITID already available."
+if [[ -z "$web_cache" ]]; then
+  echo "$1 cache with commit id $WEB_COMMITID already available."
 	# https://discourse.drone.io/t/how-to-exit-a-pipeline-early-without-failing/3951
 	# exit a Pipeline early without failing
 	exit 78
 else
-	echo "$1 cache with commit id $WEB_COMMITID was not available."
+    echo "$1 cache with commit id $WEB_COMMITID was not available."
```

Comment:
```suggestion
	echo "$1 cache with commit id $WEB_COMMITID was not available."
```

Reply:
not a good practice; if we are giving 2 tabs everywhere, then why we should give 3 tab here

Reply:
just make it consistent with rest of the code. either 2 spaces or a tab

Reply:
@saw-jan I used one tab everywhere. Does this works